js/packages/common/src/contexts/meta/loadAccounts.ts

```diff
@@ -19,6 +19,7 @@ import {
   decodeMetadata,
   getAuctionExtended,
 } from '../../actions';
+import { uniqWith } from 'lodash';
```

Comment:
I'm not crazy about adding more lodash usage - but I guess there are already debounce and other references